libvast/src/system/default_application.cpp

```diff
@@ -58,10 +58,12 @@ default_application::default_application() {
   import_ = add(nullptr, "import", "imports data from STDIN or file",
                 opts()
                   .add<caf::atom_value>("table-slice,t", "table slice type")
-                  .add<bool>("node,n",
-                             "spawn a node instead of connecting to one")
                   .add<bool>("blocking,b",
-                             "block until the IMPORTER forwarded all data"));
+                             "block until the IMPORTER forwarded all data")
+                  .add<bool>("node,N",
+                             "spawn a node instead of connecting to one")
+                  .add<size_t>("num,n",
```

Comment:
I find `num` too short for a long option. 😉

How about using `max-events` instead? That also works for import and export, describes the intent better as well. Only downside, it doesn't begin with the same letter as the short option. I think that's fine because most people still find `-n` very idiomatic for a number of events.

Reply:
Agreed.